### ISSUE TITLE
Not detecting SNES Classic / Sync Games

### ISSUE THREAD

**Issue by reverte** [ClusterM/hakchi2#735]
I flash the custom kernel, everything seems to work fine, then I try to add games to the SNES Classic, it won't recognize the console, I am using last version of Hakchi.

These are the steps I have followed:
1.- I have been able to make a kernel dump using FEL mode.
2.- I have also been able to update the custom kernel using FEL mode.
3.- Then I added some games on hakchi2.
4.- Finallay I tried to synchronize. In this step, hakchi2 does not detect the SNESC. When starting the SNES in NOT FEL mode (just power on and wait some seconds) nothing happens.

Thanks in advance.

**Comment by LeanderAT** [comment]
I also had some problems...is the driver installed?

**Comment by reverte** [comment]
Yes, it is. Otherwise I could not have flashed the kernel. Am I wrong?

**Comment by Burstaholic** [comment]
I just had this problem - after messing with it for a bit, I did `Flash custom kernel` from the menu, even though it had already been done in the initial setup, and then it worked (this time I got the "checksum failed" message, which I hadn't the first time).

I'm not sure if my having used Zadig to do the additional driver install after initial setup helped, but just that wasn't enough.

**Comment by LeanderAT** [comment]
I had some problems with the SNES Mini. First, it was not found...then I installed the driver again, it was found, I flashed it, saved a backup, , then it suddently disappeared...guides say you only have to flash it once but I had to flash the original and custom kernel more than once...then I received a checksum error...next attempt, the SNES Mini could not be found again...I tried different USB ports and then it worked after some trying and gettings another error message.

It is important NOT to use the reset button once flashing was done. When there are games on the SNES and you would like to edit them later on or add new ones, just do it in Hakchi, then connect the SNES Mini via USB, then use the power button...but do NOT use the reset button (don't hold it). I always get the message "SNES not connected" but it takes some seconds till it is found, the message goes away and games are synchronized.
